lib/graphql/schema.ts

```diff
@@ -1060,14 +1068,14 @@ export type UpdateClientInput = {
   id: Scalars['String'],
 };
 
-/** The available fields to update a Standing Order */
+/** The available fields to update a transfer */
 export type UpdateTransferInput = {
-  /** The ID of the Standing Order to update */
+  /** The ID of the transfer to update */
   id: Scalars['String'],
   /** The type of transfer to update, currently only Standing Orders are supported */
-  type: TransferType,
-  /** The amount of each Standing Order payment in cents */
-  amount: Scalars['Int'],
+  type?: Maybe<TransferType>,
```

Comment:
It's not related to this PR but rather to the corresponding backend PR, I didn't notice that we changed this 🤔 do you remember the reason why we made it optional? It seems backend will throw if it is not provided

Reply:
huh, true, this should not be optional 🤔

Reply:
It's fine, we can adjust it later if you need to have this for front-end, it's not really a big issue 👍

Reply:
@hihuz I've created a PR to fix it on backend side: https://github.com/kontist/figure-backend/pull/6504